app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
```

Comment:
- [ ] To make your project professional, kindly use the correct Readme template in your project. [CORRECT README TEMPLATE](https://github.com/microverseinc/readme-template/blob/master/README.md)